test/test_testrail_reporter.py

```diff
@@ -31,74 +33,91 @@ def setUp(self):
             ]
         }
 
-        with open('testrail.yml', 'w') as outfile:
+        with open(u'testrail.yml', u'w') as outfile:
             yaml.dump(data, outfile, default_flow_style=False)
 
     def tearDown(self):
-        os.remove('testrail.yml')
+        os.remove(u'testrail.yml')
 
     def test_load_config(self):
-        testrail_reporter = TestrailReporter('master')
+        testrail_reporter = TestrailReporter(u'master')
         empty_config = {}
 
         self.assertNotEqual(empty_config, testrail_reporter.config)
 
     def test_build_comment_for_scenario(self):
-        testrail_reporter = TestrailReporter('master')
+        testrail_reporter = TestrailReporter(u'master')
 
-        step_01 = Mock(status='passed', keyword='given')
-        step_01.name = 'step_01'
-        step_02 = Mock(status='passed', keyword='given')
-        step_02.name = 'step_02'
+        step_01 = Mock(status=u'passed', keyword=u'given')
+        step_01.name = u'step_01'
+        step_02 = Mock(status=u'passed', keyword=u'given')
+        step_02.name = u'step_02'
 
         steps = [step_01, step_02, ]
 
         mock_scenario = Mock(Scenario)
-        mock_scenario.name = 'Dummy scenario'
+        mock_scenario.name = u'Dummy scenario'
+        mock_scenario.steps = steps
+        comment = testrail_reporter._buid_comment_for_scenario(mock_scenario)
+        expected_comment = (
+            u'Dummy scenario\n'
+            u'->  given step_01 [passed]\n'
+            u'->  given step_02 [passed]')
+        self.assertEqual(expected_comment, comment)
+
+    def test_build_comment_for_scenario_with_utf8_string(self):
```

Comment:
It would be slightly more accurate to use `unicode` instead of `utf8` in this test name. The `£` character is the abstract unicode type in python by the time the code uses it. The only place `utf8` applies is in the python file's encoding.